website/src/components/SearchPage/DownloadDialog/SequenceFilters.tsx

```diff
@@ -34,19 +38,35 @@ export interface SequenceFilter {
  * Filter sequences based on certain fields that have to match, i.e. 'country == China' or
```

Comment:
nit: IMO this docstring sounds like documentation for a function rather than a class - i.e. it suggests the class will actively do this thing whereas its more a means of storing data for doing this thing in LAPIS

Reply:
https://github.com/loculus-project/loculus/pull/3883/commits/875217ae45774b10c5155ec28c67c37b77e987f2

Reply:
👍